src/AppBundle/Entity/Affiliate.php

```diff
@@ -90,9 +97,9 @@ public function getUrl()
      *
      * @param string $email
      *
-     * @return Affiliate
+     * @return self
      */
-    public function setEmail($email)
+    public function setEmail(?string $email) : self
```

Comment:
email is not nullable and in this case it's better not to allow to pass null. Better to write:
```php
public function setEmail(string $email) : self
```
in this case. Fix it for all not nullable fields.